docs/getting_started.md

```diff
@@ -57,10 +57,13 @@ auto inst_builder_ret = instance_builder
         .build();
 ```
 
-To query the available layers and extensions, use the `get_system_info()` function of `vkb::InstanceBuilder` to get a `SystemInfo` struct. It contains a `is_layer_available()` and `is_extension_available()` function to check for a layer or extensions before enabling it. It also has booleans for if the validation layers are present and if the VK_EXT_debug_utils extension is available.
+To query the available layers and extensions, get a `SystemInfo` struct from `SystemInfo::get_system_info()`. It contains a `is_layer_available()` and `is_extension_available()` function to check for a layer or extensions before enabling it. It also has booleans for if the validation layers are present and if the VK_EXT_debug_utils extension is available.
 
 ```cpp
-auto system_info = instance_builder.get_system_info();
+auto system_info_ret = vkb::SystemInfo.get_system_info();
+if (!system_info_ret) return "Error: vulkan unavailable";
```

Comment:
Perhaps you wanted to printf this and return -1? It is just example code, but a bit weird.

Reply:
Fixed. Yes, its good to have the docs reflect actual usage of the library.